src/cascadia/TerminalApp/TerminalPage.cpp

```diff
@@ -2735,4 +2754,9 @@ namespace winrt::TerminalApp::implementation
             WindowRenamer().IsOpen(false);
         }
     }
+
+    bool TerminalPage::IsQuakeWindow() const
+    {
+        return WindowName() == L"_quake";
```

Comment:
nit: maybe move `L"_quake" to the top as a `static constexpr std::wstring_view`?